Source/CompSpinalEngineTrail.cs

```diff
@@ -0,0 +1,179 @@
+﻿using RimWorld;
+using SaveOurShip2;
+using System.Text;
+using Verse;
+
+namespace TheCafFiend
+{
+    [StaticConstructorOnStartup]
+    public class CompSpinalEngineTrail : CompEngineTrail
+    {
+        public override string CompInspectStringExtra()
+        {
+            StringBuilder stringBuilder = new StringBuilder();
```

Comment:
🤔  I did some thinking about this yesterday and today and I like the "clarity" but I think I only say that because this isn't an unwieldy, long line. Figure it's better to only use var when it's a silly long line (a sort of disjointed style problem?) or better to just consistently style one way or the other?

Reply:
100% a consistent style is better.  That's part of why I think of it - it's part of our style guide at work :)